05_iigo/index.tex

```diff
@@ -347,52 +348,69 @@ \section{Accountability Cycle}
 
-Institutional-enabled-power actions in the IIGO have an associated cost. Some examples of such actions include:
+For the simulation we have defined the resources gathered by the islands to be endogenous. Hence we assume that self-organization will consume those resources and institutional-power-enabled actions in the IIGO have an associated cost. The institutional-power actions with such a cost are:
 
 
 %that is defined as a configuration parameter. The institutional-power-enabled actions of  identified to require a "computational" component are:
 
 
-%We have defined the resource to be an endogenous one, hence any computation surrounding the distribution of the resource must use up some of that resource. 
+%We have defined the resource to be an endogenous one, hence any computation surrounding the distribution of the resource must use up some of that resource.
 \begin{itemize}
-\item Calling any vote and computing the winner.
+\item President selecting a rule from the rule proposal list.
+\item President deciding the amount of individual taxation.
+\item President deciding the allocation of resources from the common pool.
+\item Speaker calling a vote and calculating the winner.
+\item Judge inspecting an island's actions.
+\item Judge inspecting an island's action history retroactively.
 \item Declaring (e.g. \textit{announcing} the result of a vote).
-\item Setting the amount of \emph{taxation} for each island.
-\item Deciding the allocation distribution for each island.
-\item Inspecting an island's action history.
+\item Holding an election.
 \item Monitoring a role.
 \end{itemize}
 
 IIGO has been designed to act in the common good. Therefore IIGO-related costs will be directly withdrawn from the common pool. Since the common pool is considered communal property of the archipelago, there are rules in place to limit how much each role is allowed to spend in order to perform its own institutional-power-enabled actions. This is the reason for defining the \emph{budget} and keeping it separate for each of the three IIGO roles.
 
-
-As a role performs institutional-enabled-power actions, the cost associated with this action is subtracted from the role's budget. A budget of zero means that the role does not have the \emph{power} to perform any of its institutional-power-enabled actions. The removal of the budget rule from the rules in play means the role is allowed to perform as many such actions as it would like (as long as those actions are not governed by other rules). 
+\begin{rule_IIGO} \label{rule:budget}
+    %Each role has the \emph{obligation} to pay the salary of amount $S$ to another if and only if the amount paid $S'$ is equal to $S$.
+    Each role has the \emph{permission} to act on an institutional-power action with an associated cost if the budget would not become negative as a result of performing the action.
```

Comment:
```suggestion
    Each role has the \emph{permission} to act on an institutionally-enabled-power action with an associated cost if the budget would not become negative as a result of performing the action.
```

Reply:
```suggestion
    Each role has the \emph{permission} to act on an institutional-power-enabled action with an associated cost if the budget would not become negative as a result of performing the action.
```

Read as (institutional-power)-enabled action. How do you read institutionally-power-enabled? Neither (institutionally-power)-enabled nor institutionally-(power-enabled) make sense to me

Reply:
you are right sorry